.github/workflows/a-team.yml

```diff
@@ -8,6 +8,52 @@ on:
       - main
 
 jobs:
+  aws_setup:
+    if: github.event.pull_request.head.repo.full_name == github.repository
+    env:
+      AWS_ACCESS_KEY_ID: ${{ secrets.ATEAM_AWS_ACCESS_KEY_ID }}
+      AWS_SECRET_ACCESS_KEY: ${{ secrets.ATEAM_AWS_SECRET_ACCESS_KEY }}
+      TF_VAR_ORG: osbuild
+      TV_VAR_REPO: automotive-ci
+      GITHUB_TOKEN: ${{ secrets.ACCESS_TOKEN }}
+
+    name: setup AWS resources
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Terraform Setup
+        uses: hashicorp/setup-terraform@v1
+        with:
+          terraform_version: 1.0.2
+
+      - name: Terraform Format
+        id: fmt
+        run: terraform -chdir=tform/aws-resources/ fmt -check
+
+      - name: Terraform Init
+        id: init
+        run: terraform -chdir=tform/aws-resources/ init
+        continue-on-error: false
+
+      - name: Terraform Validate
+        id: validate
+        run: terraform -chdir=tform/aws-resources/ validate -no-color
+
+      - name: Terraform Plan
+        id: plan
+        if: github.event_name == 'pull_request'
```

Comment:
Isn't the job run only on pull request event?

Reply:
Yes, missed this one, addressed :+1: